config.toml.example

```diff
@@ -171,37 +176,37 @@ changelog-seen = 2
 # first compiler.
 #
 # Defaults to host platform
-#build = "x86_64-unknown-linux-gnu"
+#build = <host platform>
 
 # Which triples to produce a compiler toolchain for. Each of these triples will
 # be bootstrapped from the build triple themselves.
 #
 # Defaults to just the build triple
-#host = ["x86_64-unknown-linux-gnu"]
+#host = [build]
 
 # Which triples to build libraries (core/alloc/std/test/proc_macro) for. Each of
 # these triples will be bootstrapped from the build triple themselves.
 #
 # Defaults to `host`. If you set this explicitly, you likely want to add all
 # host triples to this list as well in order for those host toolchains to be
 # able to compile programs for their native target.
-#target = ["x86_64-unknown-linux-gnu"]
+#target = host (array)
 
 # Use this directory to store build artifacts.
 # You can use "$ROOT" to indicate the root of the git repository.
 #build-dir = "build"
 
 # Instead of downloading the src/stage0.txt version of Cargo specified, use
 # this Cargo binary instead to build all Rust code
-#cargo = "/path/to/bin/cargo"
+#cargo = "<build-dir>/<host platform>/stage0/bin/cargo"
 
 # Instead of downloading the src/stage0.txt version of the compiler
 # specified, use this rustc binary instead as the stage0 snapshot compiler.
-#rustc = "/path/to/bin/rustc"
+#rustc = "<build-dir>/<host platform>/stage0/bin/rustc"
 
 # Instead of download the src/stage0.txt version of rustfmt specified,
 # use this rustfmt binary instead as the stage0 snapshot rustfmt.
-#rustfmt = "/path/to/bin/rustfmt"
+#rustfmt = "<build-dir>/<host platform>/stage0/bin/rustfmt"
```

Comment:
I think this change is intending to specify the default, but I think it's not a good idea for us to say what the default is in this case. I really would prefer that people keep the exact location of these files in the "default" out of there configuration, and really even out of their knowledge -- if folks are configuring something else, then the old suggestions here were just as good, but if you're fine with the default then you shouldn't set this at all. We already note the default is specified via src/stage0.txt just above each of these, which seems good enough to me.